src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionsHandler.cs

```diff
@@ -48,7 +48,7 @@ public async Task<LSP.CodeAction[]> HandleRequestAsync(LSP.CodeActionParams requ
             var document = context.Document;
             if (document == null)
             {
-                return Array.Empty<VSCodeAction>();
+                return Array.Empty<VSInternalCodeAction>();
```

Comment:
Does this need to return `VSInternalCodeAction` or can it just return `LSP.CodeAction`?

Reply:
It could return LSP.CodeAction definitely.  I don't think it matters though, it will just serialize as an empty array not specific to a type of code action.